README.md

```diff
@@ -21,11 +37,11 @@ code, available from the master GIT repository at:
 
    http://github.com/indigo-dc/opie/
 
-Any new code MUST follow the development guidelines detailed in the HACKING.rst
+Any new code MUST follow the development guidelines detailed in the HACKING.md
 file, and pass all unit tests. Further developer focused documentation is
 available at:
 
    http://docs.openstack.org/developer/nova/
 
 For information on how to contribute to Nova, please see the contents of the
-CONTRIBUTING.md file.
+Developer.md file.
```

Comment:
Why not CONTRIBUTING.md? This way it will show nicely on github (see https://github.com/blog/1184-contributing-guidelines)